tests/publiccloud/prepare_tools.pm

```diff
@@ -102,11 +107,16 @@ sub run {
     # Create some directories, ipa will need them
     assert_script_run("img-proof list");
     record_info('img-proof', script_output('img-proof --version'));
+    my $terraform_version = '0.14.1';
+
+    # Terraform in a container
+    my $terraform_wrapper = <<EOT;
+#!/bin/sh
+podman run -i -t hashicorp/terraform:$terraform_version \$@
```

Comment:
Do we actually need `-i`? We are doing one call and then getting the result, right?

Reply:
you right , was blindly copy/pasting

Reply:
fixed

Reply:
you can remove even the `-t`